src/component/App.js

```diff
@@ -1,12 +1,46 @@
 import './App.css';
-import DataTable from './DataTable'
+
+import firebase from 'firebase/app';
+import 'firebase/firestore';
+import 'firebase/auth';
+
+import { useCollectionData } from 'react-firebase-hooks/firestore';
+
+firebase.initializeApp({
+    apiKey: "AIzaSyBe9RfBFcXjrGtQ7jVxtrxSOWhqf28DHYk",
+    authDomain: "gwwc-data-table.firebaseapp.com",
+    databaseURL: "https://gwwc-data-table.firebaseio.com",
+    projectId: "gwwc-data-table",
+    storageBucket: "gwwc-data-table.appspot.com",
+    messagingSenderId: "924988001345",
+    appId: "1:924988001345:web:655254cb48b7162b6f4ac0",
+    measurementId: "G-TC4E5JES7F"
+})
+
+const firestore = firebase.firestore();
 
 function App() {
+
   return (
     <div className="App">
+      <p>here's the data table:</p>
       <DataTable></DataTable>
     </div>
   );
 }
 
+function DataTable() {
+    const organisationsRef = firestore.collection('organisations')
+    const query = organisationsRef.orderBy('Full Name').limit(25)
```

Comment:
With realtime database we use callbacks to get the data. Since it is a _realtime_ database, it uses some sort of sockets to emit the event when the data is updated - as such we do not need to requery the data. However, now for simplicity, I suggest we use the 'once' method, without listening for further updates. See [the doc](https://firebase.google.com/docs/database/web/read-and-write#read_data_once) for more details. Here's an example how to get the data once and log it. The code works for me locally.

```
firebase.database().ref('/organisations/').once('value').then((snapshot) => {
                const organisations = snapshot.val();
                console.log(organisations);
        });
```

Reply:
This should be wrapped in React's `useEffect` function as we are calling something asyncronous to set state. e.g.

```
const [organizations, setOrganizations] = useState([]);

useEffect(() => {
    firebase.database().ref('/organisations/').once('value').then((snapshot) => {
            setOrganizations(snapshot.val());
    });
}, [])
```